.github/workflows/prod.yml

```diff
@@ -69,6 +72,7 @@ jobs:
           sed -i -e "s|ALGOLIA_SEARCH_API_KEY_B64|$(echo -n ${{ secrets.ALGOLIA_SEARCH_API_KEY }}| base64)|g" ci/prod.yaml
           sed -i -e "s|ALGOLIA_WRITE_API_KEY_B64|$(echo -n ${{ secrets.ALGOLIA_WRITE_API_KEY }}| base64)|g" ci/prod.yaml
           sed -i -e "s|ALGOLIA_INDEX_B64|$(echo -n ${{ secrets.ALGOLIA_INDEX }}| base64)|g" ci/prod.yaml
+          sed -i -e "s|ENV_B64|$(echo -n ${{ vars.ENV }}| base64)|g" ci/staging.yaml
```

Comment:
this should be ci/prod.yml

Reply:
Updated as prod. Also fixed ci/staging.yaml variable name